tests/functional/filter.t

```diff
@@ -52,6 +55,7 @@ Using the default probabilistic subsampling, should work the same as the previou
 
   $ ${AUGUR} filter \
   >  --sequences filter/sequences.fasta \
+  >  --sequence-index filter/sequences.fasta.idx \
```

Comment:
These test files should be called `sequence_index.tsv` for consistency across documentation and workflows.